APCAppCore/APCAppCore/DataSubstrate/Model/APCSchedule+Bridge.h

```diff
@@ -33,6 +33,3 @@
  
 #import "APCSchedule.h"
```

Comment:
It seems this file is now empty, save for the license and import, can it be removed?